packages/adapter-static/index.js

```diff
@@ -1,18 +1,2 @@
-module.exports = function ({ pages = 'build', assets = 'build' } = {}) {
-	/** @type {import('@sveltejs/kit').Adapter} */
-	const adapter = {
-		name: '@sveltejs/adapter-static',
-
-		async adapt(utils) {
-			utils.copy_static_files(assets);
-			utils.copy_client_files(assets);
-
-			await utils.prerender({
-				force: true,
-				dest: pages
-			});
-		}
-	};
-
-	return adapter;
-};
+import commonjs from './index.cjs';
```

Comment:
it could be worth adding a comment here explaining why we need to use CommonJS with an ESM wrapper

Reply:
I realised there's actually no benefit to this file — removed